packages/lens/lens.1.2.5/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+maintainer: "Paolo Donadeo <p.donadeo@gmail.com>"
+authors: [ "Alessandro Strada <alessandro.strada@gmail.com>" ]
+license: "BSD-3-Clause"
+homepage: "https://github.com/pdonadeo/ocaml-lens"
+dev-repo: "git+https://github.com/pdonadeo/ocaml-lens.git"
+bug-reports: "https://github.com/pdonadeo/ocaml-lens/issues"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "ocaml"           {>= "4.10"}
+  "ppx_deriving"    {< "5.0"}
+  "ppx_tools"       {build}
+  "ppxfind"         {build}
```

Comment:
Are you sure this is still the case? It seems to conflict with your changelog

Reply:
Of course you are right, I forgot to update the opam file. I'm going to submit again. @kit-ty-kate